things/groups/service.go

```diff
@@ -6,9 +6,10 @@ import (
 
 	"github.com/mainflux/mainflux"
 	"github.com/mainflux/mainflux/internal/apiutil"
+	mfclients "github.com/mainflux/mainflux/internal/mainflux"
+	mfgroups "github.com/mainflux/mainflux/internal/mainflux/groups"
 	"github.com/mainflux/mainflux/pkg/errors"
 	"github.com/mainflux/mainflux/users/policies"
-	upolicies "github.com/mainflux/mainflux/users/policies"
```

Comment:
retain `upolicies` and remove `policies`

Reply:
why use an aliases when one is not required